src/palladio_fs/RulePackageFS.cpp

```diff
@@ -20,10 +21,10 @@ bool isRulePackageURI(const char* p) {
 		return false;
 
 	// needs to start with rpk: schema
```

Comment:
I guess with the changes the comment should now be "// needs to contain rpk: schema"